gnome/tracker/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           gobject_introspection 1.0
 
 name                tracker
 version             1.12.4
-revision            15
+revision            16
```

Comment:
is this only fixing a build failure; if so, you should not increase the "revision".

Reply:
I'll change it back then